.github/workflows/rpm-build-and-test-report.yml

```diff
@@ -0,0 +1,120 @@
+name: 'RPM Build and Test Report'
+
+on:
+  workflow_run:
+    workflows: ['RPM Build and Test']
+    types:
+      - completed
+  # for testing before landing
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  actions: read
+  checks: write
+
+jobs:
+  report-vm-1:
+    runs-on: [self-hosted, docker]
+    strategy:
+      matrix:
+        # TODO: figure out how to determine this matrix
+        distro: ['el8', 'el9', 'leap15']
+    env:
+      STAGE_NAME:
+    steps:
+      - name: Set variables
+        run: |
+            case ${{ matrix.distro }} in
+                'el8')
+                    DISTRO_NAME="EL"
+                    DISTRO_VERSION="8"
+                    ;;
+                'el9')
+                    DISTRO_NAME="EL"
+                    DISTRO_VERSION="9"
+                    ;;
+                'leap15')
+                    DISTRO_NAME="Leap"
+                    DISTRO_VERSION="15.4"
+                    ;;
+            esac
+            echo "STAGE_NAME=Build RPM on $DISTRO_NAME $DISTRO_VERSION" >> $GITHUB_ENV
```

Comment:
Perhaps.  I tried that intially with the build and test matrixes but as you can see we set a lot of variables for those.  I did the same here for consistency in approach.

Moreover this file is more-or-less just a placeholder at this point.  I need it to be on master so that I can further develop it in a PR because it won't execute in a PR until it exists in master.  This is because it runs with elevated privileges as it is a priviledge separation technique so that we are not giving repo write permissions to incoming PRs.